x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/dissect.tsx

```diff
@@ -58,7 +59,7 @@ const fieldsConfig: Record<string, FieldConfig> = {
     helpText: (
       <FormattedMessage
         id="xpack.ingestPipelines.pipelineEditor.dissectForm.appendSeparatorHelpText"
-        defaultMessage="The character(s) that separate the appended fields. Default value is {value} (an empty string)."
+        defaultMessage="Characters used to separate fields when appending two or more results together. Defaults to {value}."
```

Comment:
This description didn't make sense to me. Is it used to separate or append results?

Reply:
I dug a little deeper on this field, and the append separator is only applicable when an [append modifier](https://www.elastic.co/guide/en/elasticsearch/reference/master/dissect-processor.html#dissect-key-modifiers) is specified. I've updated the **Pattern** field to reference the key modifiers (including a link to that section of the doc page), and noted in the append separator description that it applies only if using a modifier in the pattern.